kebechet/config.py

```diff
@@ -290,13 +289,19 @@ def run(cls, configuration_file: str) -> None:
         config.from_file(configuration_file)
 
         for (
-            managers,
             slug,
             service_type,
             service_url,
             token,
             tls_verify,
         ) in config.iter_entries():
+            #cls._tls_verification(service_url, slug, verify=tls_verify)          
+            tempfile = config.download_conf_from_url(service_url, service_type)
```

Comment:
should there a new function to get data from slug and service type or something?

Reply:
Yes, you can get it from the repo. It should be always present in the root of the git repository and named `.thoth.yaml`.